src/components/views/settings/UserProfileSettings.tsx

```diff
@@ -219,11 +242,12 @@ const UserProfileSettings: React.FC<UserProfileSettingsProps> = ({
                 </Alert>
             )}
             {userIdentifier && <UsernameBox username={userIdentifier} />}
-            {externalAccountManagementUrl && (
-                <div className="mx_UserProfileSettings_profile_buttons">
+            <div className="mx_UserProfileSettings_profile_buttons">
```

Comment:
Should/Could this not use the Flex Compound component?

Reply:
which one do you mean?

Reply:
oh, there's an element-web component? I see.